File: core/core-api/src/main/java/io/say/better/domain/member/application/converter/ConnectConverter.java
package io.say.better.domain.member.application.converter;

import io.say.better.storage.mysql.domain.entity.Connect;
import io.say.better.storage.mysql.domain.entity.Member;

public class ConnectConverter {

	private ConnectConverter() {
		throw new IllegalStateException("Utility class");
	}

	public static Connect toConnect(Member educator, Member learner) {
		return Connect.builder()
				.educator(educator)
				.learner(learner)
				.build();
	}

}

File: core/core-api/src/main/kotlin/io/say/better/domain/member/application/converter/ConnectConverter.kt
package io.say.better.domain.member.application.converter

import io.say.better.storage.mysql.domains.account.entity.Connect
import io.say.better.storage.mysql.domains.account.entity.Educator
import io.say.better.storage.mysql.domains.account.entity.Learner

class ConnectConverter private constructor() {
    init {
        throw IllegalStateException("Utility class")
    }

    companion object {
        fun toConnect(
            educator: Educator,
            learner: Learner,
        ): Connect {
            return Connect(
                educator = educator,
                learner = learner,
            )
        }
    }
}
